stdlib/Distributed/src/managers.jl

```diff
@@ -540,11 +540,14 @@ end
 
 function bind_client_port(sock::TCPSocket, iptype)
     bind_host = iptype(0)
-    if Sockets.bind(sock, bind_host, client_port[])
-        _addr, port = getsockname(sock)
-        client_port[] = port
+    while true
+        if Sockets.bind(sock, bind_host, client_port[])
```

Comment:
As we've found out, this never actually returns `false` for the cases we're interested in. What's the purpose of this change?

Reply:
It's more correct here, for once we fix `Sockets.bind`

Reply:
Ok, but `bind` doesn't work that way yet (or maybe ever?). Can we leave this particular change out until we actually know how the Sockets API will change?

Reply:
Probably no, as libuv is considering changing it so that it never works this way (https://github.com/libuv/libuv/pull/2218)